roles/matrix-dimension/templates/config.yaml.j2

```diff
@@ -0,0 +1,80 @@
+# The web settings for the service (API and UI).
+# It is best to have this run on localhost and use a reverse proxy to access Dimension.
+web:
+  port: 8184
+  address: '0.0.0.0'
+
+# Homeserver configuration
+homeserver:
+  # The domain name of the homeserver. This is used in many places, such as with go-neb
+  # setups, to identify the homeserver.
+  name: "{{ matrix_domain }}"
+
+  # The URL that Dimension, go-neb, and other services provisioned by Dimension should
+  # use to access the homeserver with.
+  clientServerUrl: "https://{{ matrix_domain }}"
```

Comment:
`clientServerUrl` should use `matrix_server_fqn_matrix`, instead of `matrix_domain`. Otherwise it will generate incorrect URLs (an example page that fails because of this is the "Sticker Packs" page).